xml/ha_storage_protection.xml

```diff
@@ -809,17 +809,50 @@ Timeout (msgwait)  : 10
      </callout>
    </calloutlist>
   </step>
-  <step>
-   <para>For a two-node cluster, decide if you want predictable or random delays. For
-   other cluster setups you do not need to set this parameter.</para>
+  <step xml:id="st-ha-storage-protect-fencing-static-random">
+   <para>
+    For a two-node cluster, in case of split-brain, there will be fencing issued from
+    each node to the other as expected. To prevent both nodes from practically
+    being reset at the same time, it is recommended to apply the following fencing
```

Comment:
Do you mean that the nodes could reset almost simultaneously?

If so (and I am trying to make the minimal possible change here):

"To prevent both nodes from being reset at practically the same time"

Or does "practically" mean something different here?

Reply:
Yes, this is about preventing nodes from getting reset simultaneously. The suggested change looks good to me.